XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -1133,7 +1168,23 @@ private void toggleConsoleButton_Click(object sender, EventArgs e)
 
                     vncScreen.UseVNC = !vncScreen.UseVNC;
                     vncScreen.UserWantsToSwitchProtocol = true;

```

Comment:
This variable is never set false, so once it becomes true, it stays true for ever.